src/components/CourseInfo/CourseInfo.jsx

```diff
@@ -0,0 +1,61 @@
+import '../../App.css';
+import Button from '../../common/Button/Button';
+import { useNavigate } from 'react-router-dom';
+import getCourseDuration from '../../helpers/getCourseDuration';
+import Header from '../Header/Header';
+import formatCreationDate from '../../helpers/formatCreationDate';
+import PropTypes from 'prop-types';
+
+function CourseInfo(props) {
+	let navigate = useNavigate();
+
+	function showCourses() {
+		navigate(`/courses`);
+	}
+
+	return (
+		<div>
+			<Header userName={props.userName} />
+			<div className='CourseInfoMain'>
+				<Button
+					type='button'
+					buttonText='< Back to courses'
+					onButtonPress={showCourses}
+				/>
+				<label style={{ textAlign: 'center', fontSize: '30px' }}>
+					<b>{props.courseItem[0].title}</b>
```

Comment:
why is the first array element addressed directly?

Reply:
Because in props I have array with data